doc/catalogs.md

```diff
@@ -0,0 +1,1219 @@
+# Kowalski's collection schemas
+
+Fritz hosts a number of astronomical catalogs on its Kowalski backend.
+Below you will find a list of available catalogs and the approximate contents of
+individual entries therein (example documents per MongoDB collection).
+Note that the is no schema enforcement, so specific fields may or may not exist
+for individual documents.
+
+
```

Comment:
Maybe this is obvious for more experienced users, but I'm not sure what are these JSON examples. I guess I'm not sure what "example documents per MongoDB collection" actually means.

These catalogs include all the cross matches on Kowalski, or just a sample?

Reply:
This is everything that is available on Kowalski. It's actually not yet exposed to everyone, only to the users who have K accounts.